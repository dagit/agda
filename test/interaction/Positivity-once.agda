-- The positivity checker should not be run twice for the same mutual
-- block. (If we decide to turn Agda into a total program, then we may
-- want to revise this decision.)

{-# OPTIONS -vtc.pos.graph:5 #-}

module Positivity-once where

A : Set₁

module M where

  B : Set₁
  B = A

A = Set
