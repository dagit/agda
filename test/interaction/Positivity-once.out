(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Positivity-once (Positivity-once.agda).\n" t)
(agda2-verbose "Positivity graph: N=2 E=1\n")
(agda2-info-action "*Type-checking*" "Finished Positivity-once.\n" t)
(agda2-status-action "Checked")
(agda2-info-action "*All Goals*" "" nil)
((last . 1) . (agda2-goals-action '()))
